src/test/java/co/rsk/federate/signing/hsm/message/ReleaseCreationInformationGetterTest.java

```diff
@@ -473,7 +459,7 @@ void getTxInfoToSign_whenBatchPegoutHasPegoutTransactionCreatedEvent_returnsOk(
         LogInfo updateCollectionsLog = createUpdateCollectionsLog(senderAddress);
         logs.add(updateCollectionsLog);
 
-        Coin pegoutAmount = bridgeConstants.getMinimumPegoutTxValue();
+        Coin pegoutAmount = mock(Coin.class);
```

Comment:
Because Coin.COIN doesn't represent the total amount of the given pegout, so the value is not really what it is supposed to be. I thought it would be better to just use a mock. Anyway, this value is just symbolic for these tests.